packages/web-components/fast-foundation/src/scroller/scroller.ts

```diff
@@ -0,0 +1,241 @@
+import { attr, DOM, FASTElement, observable } from "@microsoft/fast-element";
+// TODO: the Resize Observer related files are a temporary stopgap measure until
+// Resize Observer types are pulled into TypeScript, which seems imminent
+// At that point these files should be deleted.
+// https://github.com/microsoft/TypeScript/issues/37861
+import {
+    ConstructibleResizeObserver,
+    ResizeObserverClassDefinition,
+} from "../anchored-region/resize-observer";
+import { ResizeObserverEntry } from "../anchored-region/resize-observer-entry";
+import { ScrollerControlStyles } from "./scroller.styles";
+
+declare global {
+    interface WindowWithResizeObserver extends Window {
+        ResizeObserver: ConstructibleResizeObserver;
+    }
+}
```

Comment:
@scomea I borrowed the code for resizeObserver from AnchoredRegion. I'm not sure if I did it right as I removed what I believe I didn't need.

Reply:
Would this be worth extracting to some sort of provider for reuse?

Reply:
Probably a good idea. I "borrowed" this from @scomea 's work for AnchoredRegion.